src/main/resources/view/informationView.fxml

```diff
@@ -10,24 +10,26 @@
             fx:id="parent"
             prefWidth="300"
             fx:controller="com.example.controller.InformationController"
-            id = "scrollPane">
+            id = "scrollPane" fitToWidth="true" fitToHeight="true">
 
     <stylesheets>
         <URL value="@/css/information.css"/>
     </stylesheets>
 
     <VBox spacing="5">
-        <TextFlow textAlignment="CENTER" prefWidth="270" lineSpacing="5">
+        <TextFlow textAlignment="CENTER" lineSpacing="5">
             <Text fx:id="algorithmName" id="algorithm"/>
```

Comment:
I was wondering if it's necessary to add algorithm's name here since it's displayed in comboBox. Maybe we could name this section like "Simulation state/status", WDYT? I know it's out of scope of this ticket but it shouldn't be a big change.

Reply:
I thought about it, but in our program user can change combobox content during simulation, so sometimes displayed information could be inacurate. I would stick to current version, WDYT?

Reply:
I think it's okay.